views/pages/editTags.ejs

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <% include ../partials/head.ejs %>
+
+    <script type="text/javascript" src="/utils.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<% include ../partials/menu.ejs %>
+
+<div class="container bs-docs-container">
+    <div class="col-md-12">
+        <ol class="breadcrumb" style="margin-top: 10px;">
+            <li><a href="/">Home</a></li>
+            <li><a href="/project/<%= project.id %>"><%= project.name %></a></li>
+            <li class="active">Tags</li>
```

Comment:
I feel like this screen could be the "attach" screen (not the attachTags.ejs) since you could attach or detach tags here. The previous screen feels like Tags edit screen with an odd attach to project functionality glued in. Any particular reason it is done this way? Currently if someone wants to add new tags, or edit existing ones he/she needs to navigate to a project specific tag attachment screen.

Imho this could be `Home/<project name>/Tags` edition screen with a filter to toggle visibility of unattached tags and the previous one could be broader `Home/Tags` edition screen (not linked to any particular project). What do you think?